src/index.html

```diff
@@ -15,7 +15,7 @@ <h1>jsMail</h1>
         <ul class="client-nav-items">
           <li class="client-nav-item active">INBOX</li>
           <li class="client-nav-item">SENT</li>
-          <li class="client-nav-item">DRAFTS</li>
+          <li class="client-nav-item">IMPORTANT</li>
           <li class="client-nav-item">ACTIONS</li>
```

Comment:
Thanks for the review and feedback, @cbothner! What do you mean by "clobbering"? Not sure what you mean by leaving them here...

Reply:
Clobbering here just means overwriting. Since you always render the real mailbox names “on top” of these hardcoded `<li>`s, the only way they’d be visible is if there’s an error or if the mailbox data hadn’t loaded yet. With that in mind, I would think either nothing at all or some kind of loading indicator would be a more logical thing to hardcode.

Reply:
Ok. Now that I look at this code, I realize it shouldn't even be there. It was left over and forgotten about from an earlier iteration. Deleting it now. Thanks!